src/octopeer-github/main/Logger.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Created by Maarten on 17-05-2016.
+ */
+
+/**
+ * The Logger class (singleton) is used to log events to the console.
+ * The Logger will not output debug messages if this is not explicitly requested.
+ */
+const Logger = new (class Logger {
+
+    private isDebug = false;
+
+    /**
+     * Logs a warning to the console.
+     * @param obj The object to be logged.
+     */
+    public warn(obj: any) {
+        this.log("WARN ", obj);
```

Comment:
Maybe extract these strings to contants, so they can be referd to from either context